text/0000-lifecycle-experimental-api-mode.md

```diff
@@ -0,0 +1,122 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle Experimental API Mode
+- Start Date: 2020-08-25
+- Author(s): @hone
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: [RFC#49](https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the addition of a third API mode `experimental` and extends [RFC#49](https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md).
+
+# Definitions
+[definitions]: #definitions
+
+experimental - a new API mode that is marked as experimental. Unlike supported and deprecated APIs, this may be for an unreleased API. The purpose is to gather feedback that will be used to improve the design before it's finalized. It will not carry the same promise of stability or compatibility.
+
+# Motivation
+[motivation]: #motivation
+
+In order to have a smooth transition to 1.0 of the spec, there needs to be a way for the project to test out these ideas in the wild and gather feedback. Even beyond 1.0, this desire still stands for testing out concepts in an organized fashion where the feedback can be provided. This feedback will help us improve the design before we finalize it. The quality of feedback we get from users about their real life usage of experimental APIs will help us avoid problems we may miss when approving RFCs.
+/AP
+# What it is
+[what-it-is]: #what-it-is
+
+`experimental` is a new API mode for testing out unreleased API / spec changes on their path to being finalized. An experimental API can not be `deprecated` or `supported` as defined in [RFC#49](https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md).
+
+## Lifecycle Descriptor
+
+This RFC proposes the addition of the `experimental` key in `lifecycle.toml` following the proposed schema:
+
+```
+[apis]
+[apis.buildpack]
+  experimental = ["0.6-rc1", "1.0-alpha1"]
+[apis.platform]
+  experimental = ["1.0-alpha1"]
+
+[lifecycle]
+  version = "0.10.0"
+```
+`experimental`:
+contains an array of experimental API versions
+the APIs will correspond to specs that may not be released and in branches that are still being worked on.
+the API version must be of the format: `<major>.<minor>-<alphanumeric>`, i.e. `1.0-alpha1`.
+
+Since experimental versions are defined with a different format, they will NOT automatically get upgraded to a `supported` release when the experimental API has finalized.
+
+## Lifecycle Labels
+This will extend the builder or lifecycle image with the above descriptor file will extend the label `io.buildpacks.lifecycle.apis` with the value:
```

Comment:
```suggestion
This will extend the the `io.buildpacks.lifecycle.apis` label on the builder or lifecycle image with the values contained in the above descriptor file. For example:
```